src/components/DeleteMovieModal.js

```diff
@@ -0,0 +1,19 @@
+import "./DeleteMovieModal.css";
+import React, { useContext } from "react";
+import MovieContext from "../services/MoviesContext";
+
+export default function DeleteMovieModal({handleClose, movie}){
+    const contextValue = useContext(MovieContext);
```

Comment:
you could destructure this value

```suggestion
    const { movieRepo, setMovieRepo } = useContext(MovieContext);
    
    ...
    movieRepo.deleteMovie(movie);
    setMovieRepo(movieRepo.clone());
```

Reply:
done